### ISSUE TITLE
The issue regarding classification accuracy

### ISSUE THREAD

**Issue by Radiantzy** [zcablii/SM3Det#18]
![Image](https://github.com/user-attachments/assets/f28b8e13-493c-44e5-8011-6d4e7027cc1c)

**Comment by Radiantzy** [comment]
Hello, above is my classification accuracy. I trained with a single card. I have encountered two issues:
1. Firstly, the accuracy has dropped significantly. How can I adjust it further? Currently, my batch size is 16. Can I change from fp16 to fp32 or increase gradient accumulation to improve the accuracy?
2. Secondly, there are many cases where all are Nan in the classification. Is this a normal result? Or are there any problems in certain areas? All other configurations are consistent with yours.

**Comment by zcablii** [comment]
[20240808_114547.log](https://github.com/user-attachments/files/20860680/20240808_114547.log)
you can refer to the above log to align with my experiment. Nan is normal because it eval on each of the sub-datasets individually. My batch size is 8(GPUs) x 4(per GPU bs), you may need to adjust your lr accordingly.

**Comment by Radiantzy** [comment]
Thank you for your reply. Currently, I am using 1×16. This might be the reason for the loss of precision. But if I add 32 to my computer, it will exceed the available video memory. I'll try to see if I can add fp32 or gradient accumulation to trade time for precision. Thank you again for your reply. qvq
